DOCSP-33951: atomic typos fy2024-q4 (#487)
mongodb/docs-java: source/fundamentals/enterprise-auth.txt

The following code snippets show how to specify the authentication mechanism,
using the following placeholders:

> * ``username`` - your URL-encoded principal name, for example "username%40REALM.ME"

Reviewer: ```suggestion
* ``username`` - your URL-encoded principal name, such as "username%40REALM.ME"
```
